ister_test.py

```diff
@@ -3126,7 +3126,23 @@ def validate_template_bad_version():
     except Exception:
         exception_flag = True
     if not exception_flag:
-        raise Exception("Failed to detect bad Version")
+        raise Exception("Failed to detect bad version number")
+
+
+def validate_version_template_bad_version_str():
+    """Bad validate_version_template version string"""
+    exception_flag = False
+    template = {"PartitionLayout": [],
+                "FilesystemTypes": [],
+                "DestinationType": [],
+                "PartitionMountPoints": [],
+                "Version": "20110"}
+    try:
+        ister.validate_template(template)
```

Comment:
I expect this and the num test to call the validate_version_template function directly.

Reply:
Oops, missed this.  I'll make the change.